src/index.js

```diff
@@ -5,9 +5,17 @@ let list = [];
 const inp = document.getElementById('inpId');
 const ul = document.getElementById('ulId');
 const clsBtn = document.getElementById('clsBtn');
-
+const refres = document.getElementById('refres');
 let id = 0;
-
+refres.onclick = () => {
+  window.location.reload();
+};
+refres.addEventListener('mouseover', () => {
+  refres.classList.add('fa-spin');
+});
+refres.addEventListener('mouseout', () => {
+  refres.classList.remove('fa-spin');
+});
 inp.addEventListener('keypress', (event) => {
   if (event.key === 'Enter') {
```

Comment:
- [ ] Dear @faisalakb, for status updates (completed: true / false) we need to have one new JavaScript file imported as a module. This will create modularity between your codes and help you learn how to use modules on the way. Please create a new file, add methods related to the status updates, and import it as a module. 

 - [Optional] I have noticed you have most of the changes already on the main branch, This is not a good practice, Please remember to hallway have all your changes in the new branch where you will ask for a review from. This makes it easy for the reviewer to review your changes. Also, it makes your main branch consistent.

- [ ] Dear @faisalakb, It's expected that the task object's values for `completed` will be updated upon user actions (click on the check box) and once the update happens we need to save the change in the local storage. In your case the changes are not getting saved. 
- [ ] The other thing is, Please remember all the elements of the user interface should be fully functional and the application should be completed at this stage. Hence storing and retrieving data from the local storage should be working as expected. I notice your application is not loading the stored takes while reloading.  Please make sure the application saves all the changes as they happen and uses the saved data on reload. 

https://github.com/faisalakb/Todo_List/assets/69077061/638e995b-0f3e-4916-a371-06529f5127b0

Reply:
Please check I updated all the required instructions you mentioned above.